src/fixtures/story.html

```diff
@@ -208,7 +208,7 @@
       <div class="i-amphtml-glass-pane">
       </div>
       <div class="amp-story-cta-layer">
-        <a class="i-amphtml-story-ad-link" href="https://google.com">Learn More</a>
+        <a class="i-amphtml-story-ad-link"></a>
```

Comment:
(Oooooh, I'm thinking it could be cool to hint an error when the `<meta>` tags are not present, but we can do that later.)